Marlin/planner.cpp

```diff
@@ -1219,6 +1282,17 @@ void Planner::_buffer_steps(const int32_t (&target)[ABCE], float fr_mm_s, const
   block->acceleration_steps_per_s2 = accel;
   block->acceleration = accel / steps_per_mm;
   block->acceleration_rate = (long)(accel * 16777216.0 / ((F_CPU) * 0.125)); // * 8.388608
+  #if ENABLED(LIN_ADVANCE)
+    if (block->use_advance_lead) {
+      block->advance_speed = 2000000 / (extruder_advance_K * block->e_D_ratio * block->acceleration * axis_steps_per_mm[E_AXIS]);
```

Comment:
@Sebastianv650 I assume 2,000,000 here is going to be larger for 32-bit, and based on the timer master clock.

Reply:
That's true, it's the frequeny of the clock. So it might be variable or at least different for 32bit boards.

Reply:
The number just above used for `acceleration_rate` seems to be the one we want: `((F_CPU) * 0.125)` or 1/8 of the CPU speed.